src/components/sidebar-title/SidebarTitle.stories.jsx

```diff
@@ -47,7 +47,7 @@ LeftIcon.args = {
 // With left icon and toggle
 export const IconAndToggle = ({ content, ...args }) => (
   <div style={{ width: "293px" }}>
-    <SidebarTitle {...args} icon={<IconStarOutlined></IconStarOutlined>} toggleComponent={<Toggle></Toggle>}>
+    <SidebarTitle {...args} icon={<IconStarOutlined></IconStarOutlined>} toggleComponent={<Toggle small></Toggle>}>
```

Comment:
Toggle component is part of the composition, and there is a strong dependency on SSidebarTitle, there is no need to pass it as a prop. Please move it to the component.

Reply:
Please, avoid this pattern of passing components unless it is necessary. The composition has to be done inside the component.

The icon is a different case, since there could be multiple variations of them.